toolchain/templates/include/helpers.mako

```diff
@@ -86,7 +86,7 @@ unset CRAY_ACC_MODULE
 cd "${MFC_ROOTDIR}"
 
 cat >>"${output_summary}" <<EOL
-${target.name}: $(expr $t_${target.name}_stop - $t_${target.name}_start)
+${target.name}: $(expr (($t_${target.name}_stop - $t_${target.name}_start)*1.0/1000000000) )
```

Comment:
this does not work

Reply:
I think I have fixed it in the latest commit